cmd/promote.go

```diff
@@ -17,11 +17,20 @@ import (
 	"github.com/elastic/elastic-package/internal/promote"
 )
 
+const promoteLongDescription = `Use promote command to move packages between stages in package-storage.
```

Comment:
Let's try not to reference package-storage as should be more of an implementation detail IMO (especially as we think about redesigning the registry).

```suggestion
const promoteLongDescription = `Use this command to move packages between stages in the package registry.
```